app/src/main/kotlin/com/wire/android/ui/home/conversations/model/UIMessage.kt

```diff
@@ -434,7 +434,11 @@ sealed class UIMessageContent {
             val memberNames: List<UIText>
         ) : SystemMessage(
             R.drawable.ic_info,
-            R.string.label_system_message_conversation_failed_add_members_details
+            if (memberNames.size > 1) {
+                R.string.label_system_message_conversation_failed_add_many_members_details
+            } else {
+                R.string.label_system_message_conversation_failed_add_one_member_details
+            }
```

Comment:
How about using [plural](https://developer.android.com/guide/topics/resources/string-resource#Plurals:~:text=42%20in%20English).-,example%3A,-XML%20file%20saved) xml tag in strings file ?

Reply:
Does not fit this case sadly :( since we need to only show when expanded in the UI, but this would require a refactor in `SystemMessage` component to do it.

I'm doing here the same approach we needed to use for stopping to federate, https://github.com/wireapp/wire-android-reloaded/blob/47891fbe47af5fde2d74f30baaa63aed58de1a6e/app/src/main/res/values/strings.xml#L558-L559